src/utils.js

```diff
@@ -62,7 +62,8 @@ function handleColor(name, value) {
   const color = parseColor(value);
   if (color) {
     if (value[0] === '#' && value.length === 9) {
-      return color.rgb().toString();
+      const alpha = color.alpha().toFixed(2);
+      return new Color(color.rgb().alpha(alpha)).toString();
```

Comment:
Why do we need to wrap once again with `new Color()` ?

```ts
color.rgb().alpha(...).toString()
```

Should be enough, no ?

Reply:
Yes, you are right. Thanks!